src/components/heading/__spec__.js

```diff
@@ -41,6 +47,24 @@ describe('Heading', () => {
     expect(link.props.href).toEqual('/foobar');
   });
 
+  it('renders a back link with focus support on Internet Explorer', () => {
+    const wrapper = mount(
+      <DefaultPages>
+        <Page title={
+          <Heading title='My Second Page' backLink={() => { }} />
+        }
+        >test
+          </Page>
+      </DefaultPages>)
+
+    const link = wrapper.find(Link);
+    link.find('a').simulate('mousedown');
+
+    assertStyleMatch({
+      outline: 'solid 3px #FFB500'
```

Comment:
lets use theme here to make it more future/change proof 

`outline: 'solid 3px ' + classicTheme.colors.focus`

>

Reply:
Good point! 
Updated in https://github.com/Sage/carbon/pull/2337/commits/c3c4a51be627500c74d3912caa6aef3f15818c45